# a = ascending | d = descending

ramps:

# slope

  - id: "ramp1_a"
    from: "1"
    to:   "2"
    entry_pose: [ 0.8, 1, 0.0 ]
    exit_pose:  [ 0, -1, 0.0 ]
    blind_len:  0.1
    orientation: 0
  - id: "ramp1_d"
    from: "2"
    to:   "1"
    entry_pose: [ 0, -1, 0.0 ]
    exit_pose:  [ 0.8, 1, 0.0 ]
    blind_len:  0.1
    orientation: 180.0
  - id: "rampX"
    from: "1"
    to:   "3"
    entry_pose: [ 0.8, 1, 0.0 ]
    exit_pose:  [ 0, 0, 0.0 ]
    blind_len:  0.1
    orientation: 0

# # mi
#   - id: "ramp1_a"
#     from: "1"
#     to:   "2"
#     entry_pose: [ -14, 2, 0.0 ]
#     exit_pose:  [ 0, 1, 0.0 ]
#     blind_len:  0.1
#     orientation: 90.0
#   - id: "ramp1_d"
#     from: "2"
#     to:   "1"
#     entry_pose: [ 0, 0, 0.0 ]
#     exit_pose:  [ -14, 1, 0.0 ]
#     blind_len:  0.1
#     orientation: -90.0

  # - id: "ramp1_a"
  #   from: "1"
  #   to:   "2"
  #   entry_pose: [ 2.50, 0.00, 0.0 ]
  #   exit_pose:  [ -2.50, -5.00, 0.0 ]
  #   blind_len:  5.2
  #   orientation: 0.0
  # - id: "ramp1_d"
  #   from: "2"
  #   to:   "1"
  #   entry_pose: [ -2.50, -5.00, 180 ]
  #   exit_pose:  [ 2.50, 0.00, 180 ]
  #   blind_len:  5.2
  #   orientation: 180.0

  # - id: "ramp2_a"
  #   from: "2"
  #   to:   "3"
  #   entry_pose: [ -2.50, 5.00, 180 ] 
  #   exit_pose:  [ 2.50, 0.00, 180 ]
  #   blind_len:  5.2
  #   orientation: 180.0
  # - id: "ramp2_d"
  #   from: "3"
  #   to:   "2"
  #   entry_pose: [ 2.50, 0.00, 0.0 ]
  #   exit_pose:  [ -2.50, 5.00, 0.0 ]
  #   blind_len:  5.2
  #   orientation: 0.0

  # - id: "ramp3_a"
  #   from: "1"
  #   to:   "3"
  #   entry_pose: [ 0.00, 2.50, 0.0 ]
  #   exit_pose:  [ 0.00, -2.50, 0.0 ]
  #   blind_len:  5.5
  #   orientation: 90.0
  # - id: "ramp3_d"
  #   from: "3"
  #   to:   "1"
  #   entry_pose: [ 0.00, -2.50, -90 ]
  #   exit_pose:  [ 0.00, 2.50, -90 ]
  #   blind_len:  5.5
  #   orientation: -90.0